syncd/scripts/syncd_init_common.sh

```diff
@@ -1,6 +1,20 @@
 #!/usr/bin/env bash
 
-CMD_SYNCD=/usr/bin/syncd
+CHIP_TYPE="npu"
+
+while getopts p option
+do
+case "${option}"
+in
+p) CHIP_TYPE="phy";;
+esac
+done
+
+if [ x"$CHIP_TYPE" == x"phy" ]; then
+    CMD_SYNCD=/usr/bin/physyncd
```

Comment:
where do you build physyncd? I do not see it. I suggest both use syncd name. then just provide different cmdline option.

Reply:
Yeah, something looks really wrong here on closer inspection, let me cycle back to the developer who wrote this and get some clarity. It looks like the script itself isn't even being used.

Reply:
I've reverted the changes to syncd_init_common.sh. The intent here is for syncd to be launched via a python script, not via bash. The physyncd start script is no longer sourcing syncd_init_common, because it does not use the resulting CMD (and as you pointed out, there is no /usr/bin/physyncd). It was judged more straightforward to launch from python given its support for parsing JSON. Note we may have multiple syncds running in the physyncd docker, one per gearbox phy that is defined in (JSON) configuration.

Reply:
@lguohan please re-review, re-approve, and, if possible, merge. Thanks!